sql/pr/stale-prs.sql

```diff
@@ -0,0 +1,19 @@
+-- Stale PRs
+SELECT
+  pulls.number
+, pulls.title
+, pulls.updated_at
+, current_timestamp - pulls.updated_at AS untouched_for
+, current_timestamp - max(reviews.submitted_at) AS time_since_review
+, current_timestamp - max(pull_commits.committer_date) AS time_since_push
+FROM
+  pulls
+LEFT JOIN
+  reviews ON pulls.number = reviews.pull_number
+LEFT JOIN
+  pull_commits ON pulls.number = pull_commits.pull_number
+WHERE pulls.owner = 'trinodb' AND pulls.repo = 'trino' AND pulls.state = 'open' AND NOT pulls.draft
+GROUP BY 1, 2, 3, 4
+ORDER BY untouched_for DESC
```

Comment:
We might want to duplicate this into multiple reports, sorted by different columns, to see oldest PRs but also longest "running" ones, with largest `max(reviews.submitted_at) - pulls.updated_at`. And maybe the ones waiting longest after last review (order by time_since_review desc).